zerver/openapi/python_examples.py

```diff
@@ -84,7 +84,7 @@ def add_subscriptions(client: Client) -> None:
     # {code_example|start}
     # To subscribe other users to a stream, you may pass
     # the `principals` argument, like so:
-    user_id = 26
+    user_id = 35
```

Comment:
Ugh, what does this hard coded id represent?

Reply:
In `create_user` in this file, which is run early on, `newbie@zulip.com` user is created. This example code here wants to subscribe this user to a stream, so it hard-codes the id.